src/main/resources/application.yaml

```diff
@@ -1,14 +1,15 @@
 server:
   port: 4550
 
+#If you use a database then uncomment below lines and update db properties accordingly
 management:
   endpoint:
     health:
       show-details: "always"
-      group:
-       db:
-        include: "db"
-      validate-group-membership: false
+      # group:
+      #  liveness:
+      #   include: "db"
+      # validate-group-membership: false
```

Comment:
what would this be needed for?
```suggestion
```

Reply:
the pipeline fails without,  https://github.com/hmcts/spring-boot-template/actions/runs/7085232575/job/19281067672?pr=559

org.springframework.boot.actuate.autoconfigure.health.HealthEndpointConfiguration$HealthEndpointGroupMembershipValidator$NoSuchHealthContributorException: Included health contributor 'db' in group 'db' does not exist

but I'll remove it since we are using liveness instead of db for the group